src/libs/actions/Report.ts

```diff
@@ -740,7 +738,20 @@ function openReport(
         });
     } else {
         // eslint-disable-next-line rulesdir/no-multiple-api-calls
-        API.write(WRITE_COMMANDS.OPEN_REPORT, parameters, {optimisticData, successData, failureData});
+        API.write(
+            WRITE_COMMANDS.OPEN_REPORT,
+            parameters,
+            {optimisticData, successData, failureData},
+            {
+                getConflictingRequests: (persistedRequests) =>
+                    persistedRequests.filter((request) => {
+                        if (request.command !== WRITE_COMMANDS.OPEN_REPORT) {
+                            return true;
+                        }
+                        return isEmptyObject(newReportObject) ? request.data?.reportID === reportID : false;
```

Comment:
I don't quite get why this is needed, would you mind explaining?

Reply:
updated, but @rlinoz please do let me know if you have further questions